tests/ci/common_posix_setup.sh

```diff
@@ -23,7 +23,7 @@ PLATFORM=$(uname -m)
 
 NUM_CPU_THREADS=''
 KERNEL_NAME=$(uname -s)
-if [[ "${KERNEL_NAME}" == "Darwin" ]]; then
+if [[ "${KERNEL_NAME}" == "Darwin" || "${KERNEL_NAME}" =~ .*BSD ]]; then
```

Comment:
why needed?

Reply:
Similar to Apple, OpenBSD doesn't have a /proc/cpuinfo that you can query to get the number of processor cores. So this uses the same utility available to both macOS and OpenBSD for getting that information.